### PR TITLE
Added horrible patch for Sharp Memory Display on CC13x0

### PR DESCRIPTION
**Pull request by rei-vilo**
Changed `myClock` for non trivial `myOneMsTaskTimerClock`

Added horrible patch for CC13x0

Tested against CC1352 and CC1350 LaunchPad boards

### CODE REVIEW

**Comment by rei-vilo** [comment]
@StefanSch This PR is on par with https://github.com/energia/Energia/pull/1061


@robertinant It looks like I don't have access to this repository, so I needed to create a pull request from outside.

File: libraries/LCD_SharpBoosterPack_SPI/LCD_SharpBoosterPack_SPI.cpp [hunk 1]
```diff
@@ -576,29 +576,34 @@ static void SendToggleVCOMCommand(UArg arg)
         unsigned char command = SHARP_LCD_CMD_CHANGE_VCOM;
         command |= VCOMbit;                    //COM inversion bit
 
-        if (LCDPowerMode == LCDPowerSaveOn)
```
Comment (robertinant) [inline libraries/LCD_SharpBoosterPack_SPI/LCD_SharpBoosterPack_SPI.cpp:579]: Why does this not work for the CC13x0? I rather fix the underlying code than put in a one architecture only patch.

**Comment by rei-vilo** [comment]
I don't know. The SPI calls freeze on the `SendToggleVCOMCommand` routine against the CC1350. However, `shiftOut()` works.

As the CC1352 supersedes the CC1350, digging further may not be a priority. Hence the quick and dirty patch.

**Comment by rei-vilo** [comment]
@StefanSch I've pulled the automatic low power mode, with two functions:

```

    ///
    /// @brief    PowerSave mode
    /// @details  Turn SPI module on / off
    /// @param    mode default=HIGH=SPI on, LOW=SPI off
    /// @note     AutoLowPowerMode is desactivated
    ///
    void setManualPowerMode(bool mode = HIGH);

    ///
    /// @brief    Set automatic low power mode
    /// @param    mode default=true
    /// @note     If another device on the SPI bus,
    ///           use manual setManualPowerMode() instead
    /// @note     Desactivated by setManualPowerMode()
    ///
    void setAutoLowPowerMode(bool mode = true);
```

**Pull request merged**